main.test.js

```
@@ -0,0 +1,86 @@
+const Add = require('./main')
+
+test("addition of none given arguments" , () => {
```

We should return an exception in this case not zero, zero finally is a number.